rules/promotions/microsoft_sentinel_external_alerts.toml

```diff
@@ -0,0 +1,103 @@
+- Investigate the source and context of the alert to identify any patterns or anomalies that could indicate manipulation or false positives.
+- Consult the Microsoft Sentinel investigation guide and resources tagged in the alert for specific guidance on handling similar threats.
+
+### False positive analysis
+
+- Alerts triggered by routine administrative tasks can be false positives. Identify these tasks and create exceptions to prevent unnecessary alerts.
+- Frequent alerts from known safe IP addresses or domains may not indicate a threat. Whitelist these sources to reduce noise.
+- Alerts generated by automated scripts or scheduled tasks that are part of regular operations can be excluded by setting up filters for these specific activities.
+- Non-threatening alerts from internal network scans or vulnerability assessments should be reviewed and excluded if they are part of regular security practices.
+- Alerts from test environments or sandboxed systems can be false positives. Exclude these environments from alert generation to focus on genuine threats.
+
+### Response and remediation
+
+- Contain the threat by isolating affected systems from the network to prevent further spread or data exfiltration.
+- Review and terminate any suspicious processes or sessions identified in the alert to halt ongoing malicious activities.
+- Conduct a thorough analysis of the alert details to identify any compromised accounts or credentials and reset passwords immediately.
+- Apply relevant security patches or updates to affected systems to close any vulnerabilities exploited by the adversary.
+- Restore affected systems from clean backups to ensure the integrity and security of the environment.
+- Monitor network traffic and system logs closely for any signs of recurring or related suspicious activities.
+- Escalate the incident to the security operations center (SOC) or incident response team for further investigation and to determine if additional resources are needed.
+"""
+references = ["https://docs.elastic.co/en/integrations/microsoft_sentinel"]
+risk_score = 47
+rule_id = "74147312-ba03-4bea-91d1-040d54c1e8c3"
+rule_name_override = "microsoft_sentinel.alert.properties.friendly_name"
+setup = """## Setup
+
+### Microsoft Sentinel Alert Integration
+This rule is designed to capture alert events generated by the Microsoft Sentinel integration and promote them as Elastic detection alerts.
+
+To capture Microsoft Sentinel alerts, install and configure the Microsoft Sentinel integration to ingest alert events into the `logs-microsoft_sentinel.alert-*` index pattern.
+
+If this rule is enabled alongside the External Alerts promotion rule (UUID: eb079c62-4481-4d6e-9643-3ca499df7aaa), you may receive duplicate alerts for the same Sentinel events. To avoid this, disable the External Alerts rule when using this one.
```

Comment:
same as above `consider adding a rule exception for the External Alert rule to exclude event.module:microsoft_sentinel to avoid receiving duplicate alerts from same integration ....`